ui/components/src/LineChart/LineChart.tsx

```diff
@@ -104,7 +104,7 @@ export function LineChart({
         if (onDataZoom === undefined) {
           setTimeout(() => {
             // workaround so unpin happens after click event
-            setPinTooltip(false);
+            setTooltipPinned(false);
```

Comment:
Does this need the check for the `e` being an instance of a canvas?

Reply:
yeah good question, so this one is attached to ECharts' `datazoom` [event](https://echarts.apache.org/en/api.html#events.datazoom) so we don't have to explicitly check for HTMLCanvasElement like the others that are regular React mouse events

Reply:
For a bit more context this `setTooltipPinned(false);` is only relevant when an onDataZoom handler isn't passed in and ECharts' zoom event is called (without updating the runtime time range from the panel). I'm not sure if anyone still really uses the chart that way, it's leftover from before we implemented TimeRangeProvider, but doesn't hurt to leave in just in case